README.md

```diff
@@ -192,6 +192,41 @@ $ jsonnet -J vendor -S -e 'std.manifestYamlDoc((import "mixin.libsonnet").promet
 $ jsonnet -J vendor -m files/dashboards -e '(import "mixin.libsonnet").grafanaDashboards'
 ```
 
+### Adding custom annotations to alerts
+
+Create a new file: my_custom_mixin.libsonnet with the following:
```

Comment:
Can we add a bit more of an overview what we are doing and why?

Reply:
Sure, I'll add more context in the evening.

Reply:
I added some context to the docs, is it enough? 

You should be able to modify the PR, if you see something you don't like or there's something you see that this section of the docs should include.